Cargo.toml

```diff
@@ -168,20 +168,20 @@ slab = { version = "0.4.9", optional = true } # just to get minimal versions wor
 futures-channel = { version = "0.3", optional = true }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
-env_logger = "0.10"
+env_logger = "0.11"
 hyper = { version = "1.1.0", default-features = false, features = ["http1", "http2", "client", "server"] }
 hyper-util = { version = "0.1.10", features = ["http1", "http2", "client", "client-legacy", "server-auto", "tokio"] }
 serde = { version = "1.0", features = ["derive"] }
 flate2 = "1.0.13"
-brotli_crate = { package = "brotli", version = "7.0.0" }
+brotli_crate = { package = "brotli", version = "8.0.0" }
 zstd_crate = { package = "zstd", version = "0.13" }
 doc-comment = "0.3"
 tokio = { version = "1.0", default-features = false, features = ["macros", "rt-multi-thread"] }
 futures-util = { version = "0.3.28", default-features = false, features = ["std", "alloc"] }
 rustls = { version = "0.23", default-features = false, features = ["ring"] }
 
 [target.'cfg(windows)'.dependencies]
-windows-registry = "0.4"
+windows-registry = "0.5"
```

Comment:
`windows-registry` requires `1.74` which is newer than reqwest's current MSRV `1.64`.

Reply:
By the way, `windows-registry 0.4` requires `1.74` too but it works well so far.

I'll leave this for @seanmonstar to decide.

Reply:
`cargo msrv find --min 1.70.0` never showed anything higher than 1.70.0 😅
Other dependencies already updated to 1.71 for example.